scripts/core/editor3/styles.scss

```diff
@@ -475,15 +475,43 @@ $editor-styleButton-size: 3rem;
 	padding: 10px 20px;
 }
 
+:root {
+	--preCharacterLimitt:40ch;
+}
+
 .Editor3-editor .public-DraftStyleDefault-pre {
-	background-color: rgba(0, 0, 0, 0.05);
-	font-family: 'Roboto Mono', monospace;
-	padding: 20px;
+	background-color: rgba(123, 123, 123, 0.1);
+	font-family: 'Roboto Mono', monospace !important;
+	padding: 0 2ch;
+	overflow-y: auto;
+	position: relative;
 
 	[data-block="true"] {
 		border-bottom: none;
 		margin: 0 0 0.5;
 	}
+	pre {
+		white-space: pre !important;
+		padding-right: 2ch;
+		.public-DraftStyleDefault-block {
+			white-space: inherit;
+		}
+	}
+	&.showCharacterLimit {
+		&::after {
```

Comment:
It should probably be only a single colon for pseudo-selectors like `&:after {}`, `&:before {}` shouldn't it? You can also write it inline to avoid nesting `&.showCharacterLimit:after {}`